internal/cluster/distribution/eks/ekscluster/eks.go

```diff
@@ -178,10 +178,16 @@ const (
 	DEFAULT_VPC_CIDR     = "192.168.0.0/16"
 	DEFAULT_SUBNET0_CIDR = "192.168.64.0/20"
 	DEFAULT_SUBNET1_CIDR = "192.168.80.0/20"
+	// nodePoolNameValidationRegexp = "^[a-z0-9]*$"
```

Comment:
**Question**: is this comment left in intentionally? What purpose does it serve? (If nothing, I would propose to remove it.)

Reply:
I forgot to remove it, thanks for noticing!